cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -125,6 +138,14 @@ func (o *Options) Validate(args []string) error {
 		return errors.New("file to store private key must end in '.key'")
 	}
 
+	if o.CertFileName != "" && (len(o.CertFileName) < 4 || o.CertFileName[len(o.CertFileName)-4:] != ".crt") {
```

Comment:
Well the file extension of a PKCS#1 encoded certificate is kind of debated about, some use `.pem`... i wouldn't enforce this personally.